yang/vyatta-protocols-frr-isis-v1.yang

```diff
@@ -0,0 +1,581 @@
+          description "Extended isis-id of the ISIS instance";
+      }
+    }
+  }
+
+  grouping isis {
+    container isis {
+      configd:help "IS-IS routing protocol";
+      leaf circuit-type {
+        type level;
+        configd:help "IS-type of this circuit.";
+        description "IS-type of this circuit.";
+      }
+      container hello-interval {
+        configd:help "Interval between consecutive hello messages.";
+        description "Interval between consecutive hello messages.";
+        leaf level-1 {
+          type uint32 {
+            range "1..600";
+          }
+          units "seconds";
+          configd:help "Holding time for level-1; interval will depend on multiplier.";
+          description "Holding time for level-1; interval will depend on multiplier.";
+        }
+        leaf level-2 {
+          type uint32 {
+            range "1..600";
+          }
+          units "seconds";
+          configd:help "Holding time for level-2; interval will depend on multiplier.";
+          description "Holding time for level-2; interval will depend on multiplier.";
+        }
+      }
+      container metric {
+        configd:help "Metric for this IS-IS circuit.";
+        description "Metric for this IS-IS circuit.";
+        leaf level-1 {
+          type uint32 {
+            range "0..16777215";
+          }
+          configd:help "Level-1 metric for this IS-IS circuit.";
+          description "Level-1 metric for this IS-IS circuit.";
+        }
+        leaf level-2 {
+          type uint32 {
+            range "0..16777215";
+          }
+          configd:help "Level-2 metric for this IS-IS circuit.";
+          description "Level-2 metric for this IS-IS circuit.";
+        }
+      }
+
+      container priority {
+        configd:help "Priority for Designated Router election."; 
+        description "Priority for Designated Router election.";
+        leaf level-1 {
+          type uint8 {
+            range "0..127";
+          }
+          configd:help "Level-1 priority for this IS-IS circuit.";
+          description "Level-1 priority for this IS-IS circuit.";
+        }
+        leaf level-2 {
+          type uint8 {
+            range "0..127";
+          }
+          configd:help "Level-2 priority for this IS-IS circuit.";
+          description "Level-2 priority for this IS-IS circuit.";
+        }
+      }
+
+      leaf passive {
+        type empty;
+        configd:help "Interface is in passive mode.";
+        description "Interface is in passive mode.";
+      }
+
+      container password {
+        presence "Present if a password is set for this IS interface.";
+        configd:help "Authentication password for an IS-IS area.";
+        description "Authentication password for an IS-IS area.";
+        uses isis-password;
+      }
+
+      leaf network {
+        type network-type;
+        must "(. = \"point-to-point\") or (. = \"broadcast\")";
```

Comment:
Use single quotes - 'point-to-point' - instead of escaped double quotes.  Much cleaner.

Reply:
Hi Wivory, 

We tried using single quote in this below mentioned way. 
must "(. = \'point-to-point\') or (. = \'broadcast\')";

But "configd" process crashes if we do it in this manner. 

Thanks,

Reply:
Well, I can compile this file just fine with single quotes, and I have written countless unit tests proving single quotes work in general here. Are you definitely using the correct single quote character, as we've had problems with the wrong one being used before now.  Can you tell me what error you get in configd (traceback / log error), and attach a copy of the file that causes the problem so I can test it, as if this is really failing, that's a bug that needs fixing.

Reply:
Sorry, actually we tried using backslash with single quote that's why we got the error, the screenshot is attached. 
Now we have applied the changes and found working. It is also committed.  

Thanks.

![DANOS_configd_error](https://user-images.githubusercontent.com/62253226/78357921-f1956000-75cf-11ea-81ac-92efcf60a71f.png)
![DANOS_leaf_rectified](https://user-images.githubusercontent.com/62253226/78357981-0e319800-75d0-11ea-84b9-5984b1dd4ff2.png)

Reply:
Thanks - looks much cleaner